src/chain/pushEvents.ts

```diff
@@ -0,0 +1,40 @@
+import { Event } from '@polkadot/types/interfaces';
+import * as admin from 'firebase-admin';
+
+export type PushEvent = { pattern: string; getPushData: (event: Event) => admin.messaging.TopicMessage };
+
+export const getPushEvents = ({ networkKey }: { networkKey: string }) => {
+  return [treasuryProposedEvent(networkKey), tipsNewTipEvent(networkKey)];
+};
+
+function tipsNewTipEvent(networkKey: string) {
```

Comment:
The event configs can be just plain objects, currently you can get the network from `process.env.CHAIN_NETWORK`

Reply:
7d54354 
done ✅